src/base/rss/rss_autodownloader.cpp

```diff
@@ -174,6 +175,35 @@ void AutoDownloader::removeRule(const QString &ruleName)
     }
 }
 
+QByteArray AutoDownloader::exportRulesToLegacyFormat() const
+{
+    QVariantHash dict;
+    for (const auto &rule : rules())
+        dict[rule.name()] = rule.toLegacyDict();
+
+    QByteArray data;
+    QDataStream out(&data, QIODevice::WriteOnly);
+    out.setVersion(QDataStream::Qt_4_5);
```

Comment:
Any reason you chose 4.5 specifically?

Reply:
Previous qBittorrent versions use this data stream format version so we need to use it for compatibility purposes.

Reply:
AFAIK, Qt uses most recent version unless you specify it explicitly.

Reply:
+1
I just opened v3_3_x and see that it is hardcoded there too. So we're good.